OktaLogger.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "OktaLogger"
-  s.version          = "1.3.11"
+  s.version          = "1.3.2"
```

Comment:
Shouldn't this be `1.3.12` instead of `1.3.2`?

Reply:
you're right. I had some issues with pods and testing with multiple iterations. missed cleanup